read_lyrics.py

```diff
@@ -0,0 +1,105 @@
+import pronouncing
+
+def analyze_text(myString):
+    myString = myString.replace(",", "")
+    myString = myString.replace(".", "")
+    myString = myString.replace("?", "")
+    myList = myString.split(" ")
+    running = []
+    running.append(myList[0])
+
+    idk = 0
+    same = 0
+    usedRhymes = []
+    for i in range(1, len(myList)):
+        for j in running:
+            if not usedRhymes.__contains__(j + myList[i]):
```

Comment:
I don't think you should use `__contains__` directly like this. Refer to this (link)[http://stackoverflow.com/questions/1964934/what-does-contains-do-what-can-call-contains-function] for more info. I think `in` keyword is sufficient for this purpose.